backend/src/test/java/gov/cdc/usds/simplereport/service/model/reportstream/UploadResponseTest.java

```diff
@@ -0,0 +1,51 @@
+package gov.cdc.usds.simplereport.service.model.reportstream;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+import java.util.Date;
+import java.util.UUID;
+import org.junit.jupiter.api.Test;
+
+public class UploadResponseTest {
```

Comment:
I get this test is here to increase the coverage, I dont think we should be adding it in however, an example coverage for this would be to create UploadResponse object to test the TestResultUploadService and mock it as a feign response. That will organically add coverage for the `UploadResponse` pojo

Reply:
And now that I've looked at this object for more than 2 seconds it seems like this might be because we don't use most of the values from the response.  Might just make sense to delete them?